docs/deployment.md

```diff
@@ -78,6 +78,43 @@ Make sure your `package.json` has the `"build"` and `"start"` scripts:
 
 `next build` builds the production application in the `.next` folder. After building, `next start` starts a Node.js server that supports [hybrid pages](/docs/basic-features/pages.md), serving both statically generated and server-side rendered pages.
 
+### Docker Image
+
+Next.js can be deployed to any hosting provider that supports Docker containers. This is the approach you should use when you're deploying to container orchestrators such as Kubernetes or Hashicorp Nomad, or just want to run a docker container in a single node in any cloud provider.
+
+Here is a multi-stage `Dockerfile` using `node:alpine` that you can use:
+
+```Dockerfile
+FROM node:alpine AS builder
+RUN apk add --no-cache libc6-compat
```

Comment:
Would be good to document why I would need `libc6-compat` deps

Reply:
Just added a link that describes why it might be needed. I'm just a bit in doubt if it is needed when installing dependencies, bundling the app or both. I couldn't test it right now but I know that for running the app you don't need it.